tensorflow/buildspec-tf1.yml

```diff
@@ -37,9 +37,9 @@
     BuildTensorflowCPUTrainPy3DockerImage:
       <<: *TRAINING_REPOSITORY
       build: &TENSORFLOW_CPU_TRAINING_PY3 false
-      image_size_baseline: 4899
+      image_size_baseline: 5975
       device_type: &DEVICE_TYPE cpu
-      python_version: &DOCKER_PYTHON_VERSION py3
+      python_version: &DOCKER_PYTHON_VERSION py37
```

Comment:
Please change this to `py3` to stay consistent with what already exists. We will make an organized effort to update folder structures in the future, where our PR will spend time upon iterating on any build/test failures that come up due to the folder-name changes.

Reply:
Updated